content/en/docs/Writing policies/variables.md

```diff
@@ -339,6 +339,15 @@ Variables may be defined in a `context` for consumption by Kyverno rules. This c
         value: "foo"
 ```
 
+This snippet sets a context variable to the value of `request.object.metadata.name`.
+
+```yaml  - name: defined-jmespath
+context:
+- name: objName
+  variable:
+    jmesPath: request.object.metadata.name
```

Comment:
We should mention that if `value` is not defined, `jmespath` acts on the entire context (you can reference any variable here, request.*, previously defined variables etc.)

Reply:
Captured in https://github.com/kyverno/website/pull/548/commits/41ddd7d3a431c2aff42e3c4bd1be198ee7f3a58b